src/EnergyPlus/DataHVACGlobals.cc

```diff
@@ -241,6 +241,7 @@ namespace DataHVACGlobals {
     int const CoilVRF_FluidTCtrl_Heating(34);
 
     int const CoilDX_Cooling(35);
+    int const CoilDX_SubcoolReheat(36);
```

Comment:
I think we should back away from this.  The coil isn't really another TypeOf num, just an alternate control.  The coil will need to have a flag on it that can be accessed by unitary to alert unitary of its capabilities.  Then it won't need this type anymore.